### DISCUSSION TITLE
Nightly release

### DISCUSSION THREAD

**Discussion by egoist** [tauri-apps/tauri#5453] [Ideas]
It might be a good idea to have a nightly release against dev branch, so we can try out new features as soon as they're merged, it's been a month since last release 🤔

**Comment by amrbashir** [comment]
We had a brief discussion about and it is something we want to do but unfortunately we currently lack the resources for it (human and machine). I think it is something we will want to circle back after 2.0 (probably will be a weekly releases instead of nightly.)
